config.py

```diff
@@ -20,3 +20,6 @@ class Config:
     UAA_CHECK_ENABLED = True
     CLIENT_ID = os.getenv("CLIENT_ID", "secure_message")
     CLIENT_SECRET = os.getenv("CLIENT_SECRET", "nearest.location.roll.change")
+    THREAD_DELETION_OFFSET_IN_DAYS = 365
```

Comment:
I thought this might give us a easy way of over riding in different envs. It could be 1 year but then can't be timedelta